frame/support/src/traits/messages.rs

```diff
@@ -220,3 +230,15 @@ where
 		E::footprint(O::get())
 	}
 }
+
+/// Provides information on paused queues.
+pub trait QueuePausedQuery<Origin> {
+	/// Whether this queue is paused.
+	fn is_paused(origin: &Origin) -> bool;
```

Comment:
In which cases would the origin be useful as an input here?

Reply:
The `origin` is actually the id of the queue. The two terms are used interchangeably throughout the pallet.